crates/storage/src/cache.rs

```diff
@@ -298,8 +160,8 @@ mod tests {
         assert_eq!(len, 0);
     }
 
-    #[test]
-    fn test_basic_blocking() {
+    #[tokio::test(flavor = "multi_thread")]
```

Comment:
Had to do this because of `block_on`. A part of the "weirdness"

Reply:
See comment about runtime context.